hazelcast/src/test/java/com/hazelcast/internal/metrics/impl/MetricsServiceTest.java

```diff
@@ -121,6 +123,13 @@ public void setUp() {
         metricsRegistry.registerStaticMetrics(testProbeSource, "test");
     }
 
+    @After
+    public void tearDown() {
+        if (executionService != null) {
+            executionService.shutdown();
```

Comment:
Thanks for finding and fixing it :+1: 
I'll check if I can get rid of mocking here and test `MetricsService` with just starting an instance.